pwndbg/commands/heap.py

```diff
@@ -373,7 +374,7 @@ def top_chunk(addr=None) -> None:
 @pwndbg.commands.OnlyWhenRunning
 @pwndbg.commands.OnlyWithResolvedHeapSyms
 @pwndbg.commands.OnlyWhenHeapIsInitialized
-def malloc_chunk(addr, fake=False, verbose=False, simple=False) -> None:
+def malloc_chunk(addr, fake=False, verbose=False, simple=False, dump=False) -> None:
```

Comment:
Did you test this change btw?

I am pretty sure we cannot pass this flag to the `malloc_chunk` command because we didn't add it to `parser` object above ( ? )

Reply:
I apologize, for the oversight. I am beginner in opensource. I will change it.